### PR TITLE
First draft on how to customize email-settings.

### PR DESCRIPTION
**Pull request by chriz-uniba**

### CODE REVIEW

**Comment by chriz-uniba** [comment]
# Comments

- Not tested `SECURITY_EMAIL_SUBJECT_REGISTER`
  but found here https://github.com/tu-graz-library/invenio-config-tugraz/blob/master/invenio_config_tugraz/config.py#L89-L90
- Documentation copied and inspired by https://github.com/tu-graz-library/invenio-config-tugraz

- Related to https://discord.com/channels/692989811736182844/704625518552547329/910211265429909504
  - I put it in customize, because I would have searched/expected it there.
  - I wonder whether `reference/configuration` could be merged with `customize`.

- Happy to get more inspirations, fixes, ...

File: docs/customize/index.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ InvenioRDM instance so that it integrates into their organizational environment.
 - [Styling](styling.md) - change the look and feel of InvenioRDM to match your institution.
 - [DOI registration](dois.md) - mint DOIs with your institution's prefix.
 - [Authentication](authentication.md) - integrate with your institutional authentication system.
+- [Setup sending emails](emails.md) - integrate with your institutional email system.
```
Comment (lnielsen) [inline docs/customize/index.md:13]:
```suggestion
- [Sending emails](emails.md) - integrate with your institutional email system.
```

File: mkdocs.yml [hunk 2]
```diff
@@ -45,6 +45,7 @@ nav:
       - Change theme: 'customize/styling-theme.md'
       - Change templates: 'customize/styling-templates.md'
     - Authentication: 'customize/authentication.md'
+    - Sending Emails: 'customize/emails.md'
```
Comment (lnielsen) [inline mkdocs.yml:48]:
```suggestion
    - Sending emails: 'customize/emails.md'
```

**Review by lnielsen — APPROVED** [review]

**Comment by lnielsen** [comment]
Merged as-is with two minor fixes, to get the doc going. We can add further updates in separate PRs for the more advanced cases.

**Pull request merged**